requirements.txt

```diff
@@ -8,7 +8,7 @@ checksumdir~=1.2.0
 click~=8.0.3
 contextvars~=2.4
 fastapi-jwt-auth~=0.5.0
-fastapi[all]~=0.73.0
+fastapi[all]~=0.74.0
```

Comment:
FYI: A version upgrade would be welcome to take advantage of the latest developments that simplify the update of the Swagger API documentation. But, we need to analyse the impacts...

Reply:
Ok! This was the minimum upgrade to get the "route" information from the request